### DISCUSSION TITLE
Fast inverse square root in golang

### DISCUSSION THREAD

**Discussion by yan-aint-nickname** [TheAlgorithms/Go#646] [Show and tell]
I've implemented `Fast inverse square root` in golang, I didn't see it on a website(only in Java). Is it worth it to make a PR to the repo
Gist with code is below
https://gist.github.com/yan-aint-nickname/7be20c6903e4d434a3d04d788b5fcdb1

**Comment by raklaptudirm** [comment]
Instead of using the original comments, you should replace them with something more informative so the person reading it can learn something instead of just being confused. It is worthy of a pull request.

**Comment by tjgurwara99** [comment]
Just an fyi, this already exists in https://github.com/TheAlgorithms/Go/blob/6429dfdb87ad0b2ae462f42f5307b6c0bcaca5c4/math/binary/sqrt.go
